pyaedt/generic/spisim.py

```diff
@@ -72,10 +75,12 @@ def _compute_spisim(self, parameter, out_file="", touchstone_file="", config_fil
 
         my_env = os.environ.copy()
         my_env.update(settings.aedt_environment_variables)
+
         if is_linux:  # pragma: no cover
-            if "ANSYSEM_ROOT_PATH" not in my_env:
+            if "ANSYSEM_ROOT_PATH" not in my_env:  # pragma: no cover
                 my_env["ANSYSEM_ROOT_PATH"] = self.desktop_install_dir
-            command.append("&")
+            if "SPISIM_OUTPUT_LOG" not in my_env:  # pragma: no cover
+                my_env["SPISIM_OUTPUT_LOG"] = os.path.join(out_file, generate_unique_name("spsim_out") + ".LOG")
```

Comment:
```suggestion
                my_env["SPISIM_OUTPUT_LOG"] = os.path.join(out_file, generate_unique_name("spsim_out") + ".log")
```
To follow usual behavior, shouldn't it be ".log" here instead?